File: app/src/main/java/net/squanchy/support/widget/ViewWithForeground.java
package net.squanchy.support.widget;

import android.graphics.drawable.Drawable;

public interface ViewWithForeground {

    int foregroundGravity();

    void setForegroundGravity(int foregroundGravity);

    void setForeground(Drawable drawable);

    Drawable foreground();
}

File: app/src/main/java/net/squanchy/support/widget/ViewWithForeground.kt
package net.squanchy.support.widget

import android.graphics.drawable.Drawable

interface ViewWithForeground {

    fun foregroundGravity(): Int

    fun setForegroundGravity(foregroundGravity: Int)

    fun setForeground(drawable: Drawable)

    fun foreground(): Drawable
}
